src/main/java/com/mucommander/main/AutoProcessor.java

```diff
@@ -96,12 +95,12 @@ public class AutoProcessor
      * @param configMap Map of configuration properties.
      * @param context The system bundle context.
     **/
-    public static void process(Map configMap, BundleContext context)
+    public static void process(Map<String, String> configMap, BundleContext context)
     {
         ExecutorService executor = null;
         try {
             executor = Executors.newFixedThreadPool(8);
-            var configMapLocal = (configMap == null) ? new HashMap() : configMap;
+            var configMapLocal = (configMap == null) ? new HashMap<String, String>() : configMap;
```

Comment:
how about replacing it with something like the following before the try-block:
```yaml
if (configMap == null)
    configMap = new HashMap<>();
```

Reply:
No objections. However, since we already modify AutoProcessor maybe let's just set this argument to NotNull (the only usage of this `process` method is from `muCommander.java`, and map is already enforced to be not null: https://github.com/mucommander/mucommander/blob/50749ce332462fd36d35aaea1b9e5edf04e52e65/src/main/java/com/mucommander/main/muCommander.java#L271)

Reply:
yep, good idea